app/editor/boot.js

```diff
@@ -792,6 +792,17 @@ thin.init_ = function() {
       }
     });
 
+    // Mouse wheel zoom
+    document.body.addEventListener('mousewheel', function(e){
+      var workspace = thin.core.getActiveWorkspace();
+      if(e.ctrlKey != true || !workspace) return;
+      if(e.wheelDeltaY < 0) {
+        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() - 10);
+      }else {
+        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() + 10);
+      }
+    })
```

Comment:
I'd like you to modify the code style as follows, including other similar parts.
```suggestion
      if (e.wheelDeltaY < 0) {
        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() - 10);
      } else {
        workspace.getAction().actionSetZoom(workspace.getUiStatusForZoom() + 10);
      }
    });
```